ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -35,6 +35,8 @@ presubmits:
     - dockerless-build-temporary
     rerun_command: "/test e2e-aws"
     always_run: true
```

Comment:
You need to drop this to [avoid][1]:

```
# test that the prow config is parseable
time="2018-08-14T23:10:51Z" level=fatal msg="Error loading config." error="job pull-ci-origin-installer-e2e-aws is set to always run but also declares run_if_changed targets, which are mutually exclusive" 
```

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/1191/pull-ci-openshift-release-config/285/build-log.txt

Reply:
I was wondering about that. The documentation for prow features an example that does exactly this.